judging/scripts/generate_predictions.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Runs the Python script to generate predictions.
+# It is expected that cron will run this script on a schedule.
+# See generate_predictions_local.py for more information
+#
+# The latest script is pulled from github, along with the latest tasks list containing predictions to be generated
+# The script is then run.
+
+# TODO: pull tasks list and generate_predictions_local.py script from github to local dir
```

Comment:
Note sure how Max was planning to do this. There is no git client in the Sandbox but since the repo is public we could just download what we need via `http`, as we do with the Oxford data.

Reply:
Not sure either. The mount might be updated on a regular basis.